docs/source/software/conda.rst

```diff
@@ -40,9 +40,9 @@ Miniconda3 with Python 3.7.
 4. Type ``bash Miniconda3-latest-Linux-x86_64.sh`` to start the installation.
```

Comment:
We should change `bash Miniconda3-latest-Linux-x86_64.sh` to batch mode installation `bash Miniconda3-latest-Linux-x86_64.sh -b`.  This prevents modification of `~/.bashrc` and makes steps 5 and 6 unnecessary.